cloudify_cli/config/parser_config.py

```diff
@@ -34,7 +34,9 @@
 from cloudify_cli.constants import DEFAULT_INSTALL_WORKFLOW
 from cloudify_cli.constants import DEFAULT_UNINSTALL_WORKFLOW
 
-FORMAT_INPUT_AS_YAML_OR_DICT = 'formatted as YAML or as "key1=value1;key2=value2"'
+FORMAT_INPUT_AS_YAML_OR_DICT = \
+    ('Can be provided as wildcard based paths to YAML files, '
```

Comment:
What are these `wildcard based paths`? What is their format, and how the user is supposed to know about it?